testdata/p4_16_samples_outputs/issue1334.p4-stderr

```diff
@@ -0,0 +1,3 @@
+issue1334.p4(28): [--Wwarn=mismatch] warning: -2: negative value with unsigned type
+    d(b = 2) dinst;
+          ^
```

Comment:
The strangeness here is caused by parameter substitution. Somehow, the source info for the expression is given for `d(b = 2) dinst;` instead of `x = -b;`. Again, I think this PR simply exposed this issue ...

```
control d(out bit<32> x)(bit<32> b) {
    apply {
        x = -b;
    }
}

...

    d(b = 2) dinst;

```